gobblin-admin/src/main/java/gobblin/cli/Cli.java

```diff
@@ -0,0 +1,567 @@
+/* (c) 2015 NerdWallet All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.cli;
+
+import java.io.IOException;
+import java.net.URI;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.cli.BasicParser;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.OptionGroup;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.commons.lang.NotImplementedException;
+
+import org.joda.time.Period;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+import org.joda.time.format.PeriodFormat;
+import org.joda.time.format.PeriodFormatter;
+
+import com.google.common.io.Closer;
+
+import com.linkedin.data.template.StringMap;
+import com.linkedin.r2.RemoteInvocationException;
+
+import gobblin.configuration.ConfigurationKeys;
+import gobblin.metrics.MetricNames;
+import gobblin.rest.JobExecutionInfo;
+import gobblin.rest.JobExecutionInfoArray;
+import gobblin.rest.JobExecutionInfoClient;
+import gobblin.rest.JobExecutionQuery;
+import gobblin.rest.JobExecutionQueryResult;
+import gobblin.rest.JobStateEnum;
+import gobblin.rest.Metric;
+import gobblin.rest.MetricArray;
+import gobblin.rest.QueryIdTypeEnum;
+import gobblin.rest.QueryListType;
+
+
+/**
+ * A command line interface for interacting with Gobblin.
+ * From this tool, you should be able to:
+ *  * Check the status of Gobblin jobs
+ *  * View ...
+ *
+ * @author ahollenbach@nerdwallet.com
+ */
+public class Cli {
```

Comment:
This class is too long atm. Can you split it? Suggestion: Have classes `JobCommand`, and a class `TaskCommand`.